data/en/test_radio_optional_with_mandatory_other.json

```diff
@@ -0,0 +1,53 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "A questionnaire to demo radio field Other input.",
```

Comment:
@MIslam92 why does this have a description but others don't? Should probably keep consistent either way